### PR TITLE
[HOLD} Some final polish before releasing v2.

### PR DESCRIPTION
**Pull request by drichelson**
See green items here: https://catamorphic.quip.com/Ggk6AbylpJBf for details on what is included.

### CODE REVIEW

File: ldclient.go [hunk 1]
```diff
@@ -93,43 +91,45 @@ func MakeCustomClient(apiKey string, config Config, waitFor time.Duration) (*LDC
 		config.PollInterval = 1 * time.Second
 	}
 
-	requestor := newRequestor(apiKey, config)
```
Comment (drichelson) [inline ldclient.go]: I moved things around in this initialization code so it flows better for offline/LDD mode.

**Comment by jkodumal** [comment]
👍

**Comment by jkodumal** [comment]
Sorry- I missed the hold. FWIW, it looks good to me?

**Comment by drichelson** [comment]
Thanks for looking.. I think these changes are worth merging- they pass integration harness locally.

**Pull request merged**